frontends/web/src/routes/settings/components/advanced-settings/explorer-dialog.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright 2023 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { Dispatch, useEffect, useState } from 'react';
+import { Dialog, DialogButtons } from '../../../../components/dialog/dialog';
+import { Toggle } from '../../../../components/toggle/toggle';
+import { Button, Input } from '../../../../components/forms';
+import { setConfig } from '../../../../utils/config';
+import { socksProxyCheck } from '../../../../api/backend';
+import { alertUser } from '../../../../components/alert/Alert';
+import { TConfig, TBitcoinExplorerConfig, TBitcoinConfig } from '../../advanced-settings';
+import { debug } from '../../../../utils/env';
+
+type TProps = {
+    open: boolean;
+    bitcoinConfig?: TBitcoinConfig;
+    onCloseDialog: () => void;
+    onChangeConfig: (config: any) => void;
+    handleShowRestartMessage: Dispatch<boolean>;
+}
+
+export const ExplorerDialog = ({ open, bitcoinConfig, onCloseDialog, onChangeConfig, handleShowRestartMessage }: TProps) => {
+  const [explorerURL, setExplorerURL] = useState<string>();
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    if (bitcoinConfig?.explorer) {
+      setExplorerURL(bitcoinConfig.explorer.explorerURL);
+    }
+  }, [bitcoinConfig]);
+
+
+  const handleSetExplorerButton = async () => {
+    if (!bitcoinConfig?.explorer || explorerURL === undefined) {
+      // FIXME: this looks like we avoid handeling a problem here
+      return;
+    }
+    const explorer = bitcoinConfig.explorer;
+    explorer.explorerURL = explorerURL.trim();
+
+    // NOTE: I guess its good to pingthe server to see if it works..is socksProxyCheck ok for that ?
```

Comment:
I actually just left this in here after copy/pasting from `enable-proxy` component, check if this is correct and remove comment (it does work)